consensus/spos/interface.go

```diff
@@ -96,6 +98,7 @@ type SubroundsFactory interface {
 
 //WorkerHandler represents the interface for the SposWorker
 type WorkerHandler interface {
+	io.Closer
```

Comment:
shouldn't we use our own interface ? instead of io.Closer

Reply:
yep, removed it from all modified interfaces